src/trackForMutations.js

```diff
@@ -1,26 +1,36 @@
-export default function trackForMutations(isImmutable, obj) {
-  const trackedProperties = trackProperties(isImmutable, obj);
+export default function trackForMutations(isImmutable, ignore, obj) {
```

Comment:
I suppose we could pass the single "options" object here as well, although I could envision future cases where there isn't always a 1-to-1 relationship between middleware "options" and args needed for `trackForMutations`.